src/components/PostList/PostList.tsx

```diff
@@ -1,3 +1,17 @@
 import React from 'react';
+import { Post } from '../../types/post';
+import { PostInfo } from '../PostInfo';
 
-export const PostList: React.FC = () => <>Put the list here</>;
+type Props = {
+  posts: Post[];
+};
+
+export const PostList: React.FC<Props> = ({ posts }) => {
+  return (
+    <div className="PostList">
+      {posts.map((post: Post) => (
```

Comment:
The type is well known to TypeScript, so you don't need to apply any types here. Try to make use of TypeScript's type inference capabilities where possible.